command_line/cosym.py

```diff
@@ -232,6 +234,19 @@ def _apply_reindexing_operators(self, reindexing_ops, subgroup=None):
                 )
             )
             refl["miller_index"] = cb_op.apply(refl["miller_index"])
+        # Allow for the case where some datasets are filtered out.
+        if len(reindexing_ops) < len(self._experiments):
+            to_delete = []
+            for i in range(len(self._experiments)):
+                if i not in unique_ids:
+                    to_delete.append(i)
+            to_delete.sort(reverse=True)
+            for idx in to_delete:
```

Comment:
It _might_ be slightly cleaner to sort inline (but this is just personal preference). It's not entirely obvious to me why the `reverse=True` is necessary here?
```suggestion
            for idx in sorted(to_delete):
```

Reply:
Thanks, the reason for `reverse=True` is to avoid issues when deleting in-place multiple experiments, reflections from the lists.